packages/babel-types/src/definitions/es2015.js

```diff
@@ -19,7 +19,13 @@ defineType("AssignmentPattern", {
   fields: {
     ...patternLikeCommon,
     left: {
-      validate: assertNodeType("Identifier", "ObjectPattern", "ArrayPattern"),
+      validate: assertNodeType(
+        "Identifier",
+        "ObjectPattern",
+        "ArrayPattern",
+        "MemberExpression",
+        "CallExpression",
```

Comment:
How did `CallExpression` get into this?

Reply:
@jridgewell 
```js
[_destructureSet(MEMBER) = _VALUE] = ARR
```

Reply:
That's invalid, isn't it? Chrome throws a syntax error.

Reply:
Yes you are right. my bad. fixed it